newIDE/app/src/MainFrame/index.js

```diff
@@ -2663,33 +2654,42 @@ const MainFrame = (props: Props) => {
 
   const showLoader = isLoadingProject || previewLoading;
 
+  const shortcutMap = useShortcutMap();
+  const buildMainMenuProps = {
+    i18n: i18n,
+    project: state.currentProject,
+    recentProjectFiles: preferences.getRecentProjectFiles(),
+    shortcutMap,
+    isApplicationTopLevelMenu: false,
+  };
+  const mainMenuCallbacks = {
+    onChooseProject: () => openOpenFromStorageProviderDialog(),
+    onOpenRecentFile: openFromFileMetadataWithStorageProvider,
+    onSaveProject: saveProject,
+    onSaveProjectAs: saveProjectAs,
+    onCloseProject: askToCloseProject,
+    onCloseApp: closeApp,
+    onExportProject: () => openExportDialog(true),
+    onCreateProject: () => openCreateProjectDialog(true, null),
+    onCreateBlank: () => setNewProjectSetupDialogOpen(true),
+    onOpenProjectManager: () => openProjectManager(true),
+    onOpenHomePage: openHomePage,
+    onOpenDebugger: openDebugger,
+    onOpenAbout: () => openAboutDialog(true),
+    onOpenPreferences: () => openPreferencesDialog(true),
+    onOpenLanguage: () => openLanguageDialog(true),
+    onOpenProfile: () => openProfileDialogWithTab('profile'),
+    onOpenGamesDashboard: () => openProfileDialogWithTab('games-dashboard'),
+    setElectronUpdateStatus: setElectronUpdateStatus,
+  };
+
   return (
     <div className="main-frame">
```

Comment:
what's this className for?
If it's important, maybe a small comment explaining why I shouldn't remove it

Reply:
Adding a comment